adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/model/AdapterConfig.java

```diff
@@ -8,6 +8,8 @@
 
 public class AdapterConfig {
 
+    private long id;
```

Comment:
Where comes the id from? For what do we need it? AdapterConfig = ExecutionConfig, DatasourceConfig = stored config that leads to AdapterConfig for execution

Reply:
I think i only messed up with the naming in the event (it should be adapterconfigId instead of datasourceId).

If i got that correct, then the pipeline config is referring to the adapter config id.

I only need the adapter config id to search corresponding pipeline configs in the transformation database.
Everything keeps on being the same, except that the transformation service knows which pipeline configs 
to extract from its database in order to get the transformations functions.

The datasource id will be still passed so the transformation service knows where to get the data from (when using thin event).

So the execution plan for thin event would be something like this:
1. scheduler triggers adapter import
2. adapter sends datasource link to data and corresponding adapter config id to transformation service (via event)
3. transformation gets data from send data source link
3. transformation service searches all pipeline configs that refer to the adapter cofnig id
4. transformation executes transformation functions of all found pipelines on the data from datasourcelink
5. transformation sends event to notification service (which handles notifications...)
5. transformation stores all transformation results to the new storage service (via Event)
6. the data can be accessed on storage service (via HTTP-GET)

Reply:
> I think i only messed up with the naming in the event (it should be adapterconfigId instead of datasourceId).
> If i got that correct, then the pipeline config is referring to the adapter config id.

Maybe our structure and naming are a bit misleading. Conceptually, an adapterConfig refers to a stateless data import and does not have an id. 
A datasourceConfig, on the other hand, refers to the datasource itself for which multiple data imports (using an adapter config) can be performed.  
The pipelines, therefore, do not have a reference to adapterConfigs but to data sources, which are persisted in the adapter service as datasourceConfig. 
That means your current naming in the event (datasourceId) is correct. In an event about imported data, we need the information to which data source (NOT adapterConfig = data import) the data belongs. 
In another(!) pull request, we should probably rename adapterConfig to dataImportConfig or refactor the adapter structure altogether to make it less confusing. 
Apart from that, I think your execution plan looks good and it is probably just a misconception about the naming of the concepts. 
If what I am writing does not make sense to you, we could talk about that in meet or something, just drop me a message in gitter if that would help you.